README.md

```diff
@@ -6,7 +6,7 @@
 
 ## Considerations
 
-The functions run in **Serverless Framework** and use **Windows** like OS.
```

Comment:
now we don't have functions in windows?

Reply:
No, I found a solution for it, I am trying  to standardize the Function OS

Reply:
Only Linux